universal-application-tool-0.0.1/app/auth/ProfileFactory.java

```diff
@@ -50,10 +50,12 @@ public CiviFormProfile wrapProfileData(CiviFormProfileData p) {
     return new CiviFormProfile(dbContext, httpContext, p);
   }
 
-  private CiviFormProfileData create(Roles role) {
+  private CiviFormProfileData create(Roles[] roleList) {
```

Comment:
Not sure this is the right place for it, but it'd be great to leave some comments in the code (similar to the ones in the PR) explaining this fix since it's potentially non-obvious.

Reply:
Done